core/core.go

```diff
@@ -54,6 +57,9 @@ func NewCore(db ethdb.Database, config *Config, isLocalBlock func(block *types.H
 	appendQueue, _ := lru.New(c_maxAppendQueue)
 	c.appendQueue = appendQueue
 
+	proccesingCache, _ := lru.NewWithExpire(c_processingCache, time.Second*60)
```

Comment:
Is this redundant with `appendQueue`?

Reply:
This is a different queue, that keeps track of the blocks that are in the process of being appended